src/vs/workbench/parts/search/electron-browser/search.contribution.ts

```diff
@@ -674,6 +674,12 @@ configurationRegistry.registerConfiguration({
 			type: 'boolean',
 			default: false,
 			description: nls.localize('search.usePCRE2', "Whether to use the PCRE2 regex engine in text search. This enables using some advanced regex features like lookbehind and backreferences. However, not all PCRE2 features are supported - only features that are also supported by JavaScript.")
+		},
+		'search.actionsPosition': {
+			type: 'string',
+			enum: ['auto', 'right'],
+			default: 'auto',
```

Comment:
While I personally prefer `right` I decided to preserve behavior that was default in 1.29 because my issue (filled even before 1.29 release) got 0 👍  and only two supporting comments.

In any way it may be discussed and changed if maintainers are OK with it.

Reply:
I've already opened a PR to be address this. Thanks for making this configurable! It's been plaguing me last few days.

Reply:
We've discussed this a bunch already, I'm not going to change the default because 'right' doesn't make sense to me for wide views on large monitors. The setting is here for your to customize however you want.